charts/camunda-platform/charts/identity/templates/deployment.yaml

```diff
@@ -22,6 +22,10 @@ spec:
     spec:
       imagePullSecrets:
         {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}
+      initContainers:
+        {{- with .Values.extraInitContainers }}
+        {{- tpl (toYaml . ) $ | nindent 8  }}
```

Comment:
For this case, I'd suggest to use this `{{- tpl (.Values.extraInitContainers | default list | toYaml | nindent 8) $ }}` instead of `with ...`.

- it sets the default value to an empty list, so it's clear that the value has been rendered, not just missing.
- it's easier to read sequentially.
- it has fewer lines (1 instead of 3).
- it follows the same patterns we have, like `imagePullSecrets`.